src/workerd/util/sqlite-test.c++

```diff
@@ -663,6 +663,36 @@ KJ_TEST("SQLite write row counters (basic)") {
   }
 }
 
+KJ_TEST("SQLite read/write row counters (large row insert)") {
+  auto dir = kj::newInMemoryDirectory(kj::nullClock());
+  SqliteDatabase::Vfs vfs(*dir);
+  SqliteDatabase db(vfs, kj::Path({"foo"}), kj::WriteMode::CREATE | kj::WriteMode::MODIFY);
+
+  db.run("CREATE TABLE large_things (id INTEGER PRIMARY KEY, large_value TEXT)");
+
+  // SQLite's default page size is 4096 bytes
```

Comment:
Can you add an assertion for the default page size? It seems  `pragma_page_size() ` is available.

Reply:
Added!